internal/config/config.go

```diff
@@ -56,6 +61,15 @@ type Settler struct {
 	BatchSize int `yaml:"batch_size" default:"200"`
 }
 
+type Distributor struct {
+	// The number of demotions that trigger a slashing.
+	MaxDemotionCount int `yaml:"max_demotion_count"`
+	// The required number of qualified Nodes
```

Comment:
What is qualified node for?

Reply:
The number of qualified nodes is for finding nodes that meet the criteria for distribution.